Remotion/ObjectBinding/Web.ClientScript/Scripts/BocList.ts

```diff
@@ -704,26 +704,38 @@ class BocList
     realTable.style.marginTop = (fakeTableHeadContainerHeight * -1) + 'px';
   }
 
-  private static FixHeaderPosition(tableContainer: HTMLElement, scrollableContainer: HTMLElement): void
-  {
-    const fakeTableHeadContainer = tableContainer.querySelector(':scope > div.bocListFakeTableHead')!;
-    let scrollLeft = scrollableContainer.scrollLeft;
-    const previousScrollLeft = parseFloat(scrollableContainer.dataset.bocListPreviousScrollLeft!)
-    const fakeTableHeadScrollLeft = fakeTableHeadContainer.scrollLeft;
-
-    const hasScrollMoveFromScrollbar = previousScrollLeft !== scrollLeft;
-    const hasScrollMoveFromColumnHeader = previousScrollLeft !== fakeTableHeadScrollLeft;
-    if (!hasScrollMoveFromScrollbar && !hasScrollMoveFromColumnHeader)
-      return;
+  private static FixHeaderPosition(tableContainer: HTMLElement, scrollableContainer: HTMLElement): void {
+      const fakeTableHeadContainer = tableContainer.querySelector(':scope > div.bocListFakeTableHead')!;
+      let scrollLeft = scrollableContainer.scrollLeft;
+      const previousScrollLeft = parseFloat(scrollableContainer.dataset.bocListPreviousScrollLeft!)
+      const fakeTableHeadScrollLeft = fakeTableHeadContainer.scrollLeft;
 
-    scrollLeft = hasScrollMoveFromColumnHeader ? fakeTableHeadScrollLeft : scrollLeft;
-    scrollableContainer.dataset.bocListPreviousScrollLeft = scrollLeft.toString();
+      const hasScrollMoveFromScrollbar = previousScrollLeft !== scrollLeft;
+      const hasScrollMoveFromColumnHeader = previousScrollLeft !== fakeTableHeadScrollLeft;
+      if (!hasScrollMoveFromScrollbar && !hasScrollMoveFromColumnHeader)
+          return;
 
-    if (hasScrollMoveFromColumnHeader)
-      scrollableContainer.scrollLeft = scrollLeft;
+      scrollLeft = hasScrollMoveFromColumnHeader ? fakeTableHeadScrollLeft : scrollLeft;
+      scrollableContainer.dataset.bocListPreviousScrollLeft = scrollLeft.toString();
 
+      if (hasScrollMoveFromColumnHeader)
+      {
+          scrollableContainer.scrollLeft = scrollLeft;
+          if (scrollableContainer.scrollLeft !== scrollLeft){
```

Comment:
thanks for explaining the updated logic. I also tested it and it works so it's perfectly fine that I don't get it completely.

I also tested in NovaGray. Looks fine, there, too.